docs/diagrams/UnmarkActivityDiagram.puml

```diff
@@ -0,0 +1,11 @@
+@startuml
+start
+:User executes command;
+
+if () then ([is marked])
+    :unmark event;
```

Comment:
Maybe we can capitalise the first letters for standardisation

Reply:
sounds good, its done!